client/src/components/CourseInfo/CourseInfo.tsx

```diff
@@ -61,19 +66,25 @@ const Description = ({ text }: { text: string }) => {
   }, [text, paragraphRef.current]);
 
   return (
-    <button
+    <Button
+      variant="tonal"
+      color="tertiary"
+      ripple={false}
       className={classnames(
         ...BentoBase,
-        "w-full",
-        "mb-4",
-        "text-sys-on-tertiary-container"
+        "w-full mb-4 min-h-[5rem]",
+        clampable && "pointer-events-auto"
       )}
       onClick={() => setShowMore(!showMore)}
     >
       <p ref={paragraphRef} className={showMore ? "" : "line-clamp-3"}>
         {text}
       </p>
-      <p>{clampable && (showMore ? "LESS" : "MORE")}</p>
-    </button>
+      {clampable && (
+        <p className="font-medium uppercase text-sm mt-2 text-tertiary">
+          {showMore ? "Less" : "More"}
+        </p>
+      )}
+    </Button>
```

Comment:
This button stays grayed after clicking on it until you click away
![image](https://user-images.githubusercontent.com/70448914/222472437-fa2f56eb-2fb4-4329-88ba-3352a8b3ead3.png)